templates/index.html

```diff
@@ -4,7 +4,7 @@
 {% block masthead %}
 <h1 class="ui inverted header">
     <div>
-        <img class="ui centered image" src="assets/img/rust-logo-128x128-blk-v2-invert.png" alt="" width="128" height="128">
+        <img class="ui centered image" src="assets/img/rust-logo-blk-invert.svg" alt="" width="128" height="128">
```

Comment:
Let's remove the now unused `rust-logo-128x128-blk-v2-invert.png`?

Reply:
Done in https://github.com/rust-gamedev/arewegameyet/commit/5a99099d96e89b9a5dd47526481373c4d7760dbb